pallets/accounts/src/lib.rs

```diff
@@ -159,6 +159,12 @@ pub struct EthLock<TBalance, TBlockNum> {
 	pub amount: TBalance,
 	/// Clamor Block Number in which the locked FRAG tokens was "effectively transfered" to the Clamor Blockchain
 	pub block_number: TBlockNum,
+	/// The block number when FRAG were locked the first time
+	pub first_lock_block_number: TBlockNum,
```

Comment:
no. `block_number` is updated also with `unlock` event. So, in case the user **lock --> unlock** (both on Ethereum) and then he withdraws on Clamor, we would have lost the block number when the lock happened and so not able to mint the correct amount of Tickets and NOVA.
Is this reasoning correct?